src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridgeMeterRegistryProducer.java

```diff
@@ -10,17 +10,22 @@
 import io.micrometer.prometheus.PrometheusConfig;
 import io.micrometer.prometheus.PrometheusMeterRegistry;
 import io.micrometer.prometheus.PrometheusNamingConvention;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 import javax.annotation.PostConstruct;
 import javax.enterprise.inject.Produces;
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
 /**
- * Used for scraping and reporting metrics in Prometheus format
+ * Scrapes Quarkus http server related metrics and the Kafka related metrics coming through JMX collector
```

Comment:
This sentence is good for the `scrape()` method but should be removed from here. I added a suggestion later.